app/records/converters.py

```diff
@@ -4,4 +4,6 @@
 class IDConverter(StringConverter):
     """Converter used to extract id's from a URL."""
 
-    regex = r"([ACDFN][0-9]{1,8}|[a-f0-9]{8}-?([a-f0-9]{4}-?){3}[a-f0-9]{12}(_[1-9])?)"
+    # This should be the correct regex but some of the GUIDs are not valid
+    # regex = r"([ACDFN][0-9]{1,8}|[a-f0-9]{8}-?([a-f0-9]{4}-?){3}[a-f0-9]{12}(_[1-9])?)"
+    regex = r"([ACDFN][0-9]{1,8}|[a-f0-9]{4,8}-?([a-f0-9]{4}-?){3}[a-f0-9]{12}(_[1-9])?)"
```

Comment:
> The data is bad Response id should be `6556c5f5-0f02-4d03-9b5c-8ec7973794b7` and not `c5f5-0f02-4d03-9b5c-8ec7973794b7`

I mentioned the data is bad and to ignore, did you spot a GUID that is not compatible with the Regex?

Reply:
Yes - I added some invalid GUIDs I found to the tests